### PR TITLE
Test fixes for win7 and win8

### PR DESCRIPTION
**Pull request by bingbing8**
1. Test fixes for win7 and win8
2. Added onecore parameter to Start-OpenSSHPackage

### CODE REVIEW

File: regress/pesterTests/Authorized_keys_fileperm.Tests.ps1 [hunk 1]
```diff
@@ -27,8 +27,9 @@ Describe "Tests for authorized_keys file permission" -Tags "CI" {
         $opensshbinpath = $OpenSSHTestInfo['OpenSSHBinPath']
         Remove-Item -Path (Join-Path $testDir "*$sshLogName") -Force -ErrorAction SilentlyContinue        
         $platform = Get-Platform
-        $skip = ($platform -eq [PlatformType]::Windows) -and ($PSVersionTable.PSVersion.Major -le 2)
-        if(($platform -eq [PlatformType]::Windows) -and ($psversiontable.BuildVersion.Major -le 6))
+        #skip on win7 because Set-Acl failed due to the task schedular (*-ScheduledTask) cmdlets does not exist on win7 and win8        
+        $skip = ($platform -eq [PlatformType]::Windows) -and ([Environment]::OSVersion.Version.Major -le 6) -and ([Environment]::OSVersion.Version.Minor -le 2)
```
Comment (bagajjal) [inline regress/pesterTests/Authorized_keys_fileperm.Tests.ps1]: Is it And / OR for the last two conditions
([Environment]::OSVersion.Version.Major -le 6) -or ([Environment]::OSVersion.Version.Minor -le 2)

File: regress/pesterTests/Hostkey_fileperm.Tests.ps1 [hunk 2]
```diff
@@ -22,7 +22,7 @@ Describe "Tests for host keys file permission" -Tags "CI" {
         $script:logNum = 0
         Remove-Item -Path (Join-Path $testDir "*$logName") -Force -ErrorAction SilentlyContinue        
         $platform = Get-Platform
-        $skip = ($platform -eq [PlatformType]::Windows) -and ($PSVersionTable.PSVersion.Major -le 2)
+        $skip = ($platform -eq [PlatformType]::Windows) -and ([Environment]::OSVersion.Version.Major -le 6) -and ([Environment]::OSVersion.Version.Minor -lt 2)
```
Comment (bagajjal) [inline regress/pesterTests/Hostkey_fileperm.Tests.ps1:25]: same comment.

File: regress/pesterTests/Authorized_keys_fileperm.Tests.ps1 (hunk 1 above)
Comment (bingbing8) [reply regress/pesterTests/Authorized_keys_fileperm.Tests.ps1]: and. 6.2 is win8. 6.1 is win7

File: regress/pesterTests/Hostkey_fileperm.Tests.ps1 (hunk 2 above)
Comment (bingbing8) [reply regress/pesterTests/Hostkey_fileperm.Tests.ps1:25]: and

**Review by bagajjal — APPROVED** [review]

**Pull request merged**